habitat-baselines/habitat_baselines/config/rearrange/rl_hierarchical_oracle_nav.yaml

```diff
@@ -13,6 +13,10 @@ defaults:
   - _self_
 
 habitat:
+  task:
+    actions:
+      oracle_nav_action:
```

Comment:
Hi Andrew! Could you please clarify what is the difference between oracle_nav and teleporting actions? Thank you!

Reply:
Oracle nav means the navigation is simulated but takes the shortest path. Teleporting actions instantaneously moves the robot around.